src/helpers/search.js

```diff
@@ -76,8 +84,20 @@ export class SuggestionsHelper {
   getSuggestions() {
     return this.suggestions;
   }
+
 }
 
+const localeCompareWithYear = (a, b) => {
```

Comment:
Can it be done with moment()?

Reply:
I don't see how moment can help here.
In the line 46 it can help, but it will be too complicate (see comment)